README.md

```diff
@@ -0,0 +1,178 @@
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+
+### Install
+
+*Install this project with:
+- any code editor
+- Node.js
+
+### Usage
+
+To run the project, execute the following command:
+
+you just have to double click on index.html file
+
+### Run tests
+
+To run tests, run the following command:
+
+you can run and verify the correct functioning of the linters from the command console using:
+npx stylelint "**/*.{css,scss}"
+
+to run the project you just have to double click on the file index.html
+
+### Deployment
+
+You can deploy this project using:
+
+-git
+-github
+-the text editor of your choice
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="Fernando Montes"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Fernando426](https://github.com/Fernando426)
+- Twitter: [@Fernando_426_](https://twitter.com/Fernando_426_)
+- LinkedIn: [Fernando Montes](https://www.linkedin.com/in/fernando-jose-montes-linares-523822270/?originalSubdomain=ve)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[interactive content]**
+- [ ] **[new sections]**
+- [ ] **[collaboration options]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+With your help we can keep moving forward, thank you!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, I would like to invite you to show your support for the growth and development of professional Web programming, any contribution, comment or suggestion will be very well received!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to first thank the team and the platform as such at Microverse for making this fun, enjoyable and useful phase of a whole path of preparation towards world-class professionalism and training. also to the people, coding partners, guides and everyone who in one way or another has contributed to the advancement of the entire program. thank you so much!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[When will the new features be ready?]**
+
+  - [you can be sure that we are working on it every day, we will be happy to inform you as soon as they are available!]
+
+- **[Why are updates slow?]**
+
+  - [we make sure that new features are fully operational before making them available to you, we want you to always have the best user experience]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix your MIT license link. It is broken at this moment. The LICENSE file you're referencing does not exist.

![license](https://github.com/Fernando426/Hello_World/assets/26597432/fef01279-8a2e-49a7-81df-335c7ef6b170)